media-libs/ctl/ctl-1.5.2-r3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="AMPAS' Color Transformation Language"
+HOMEPAGE="https://github.com/ampas/CTL"
+SRC_URI="https://github.com/ampas/CTL/archive/${P}.tar.gz"
+S="${WORKDIR}/CTL-ctl-${PV}"
+
+LICENSE="AMPAS"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc64 ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="media-libs/ilmbase:=
+	media-libs/tiff:=
+	!media-libs/openexr_ctl"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
```

Comment:
Hm, there's not even a dependency on openexr here? How are you restricting it?

Reply:
There was an upstream change reported yesterday in https://github.com/ampas/CTL/issues/100#issuecomment-1315415301 which makes dependency on OpenEXR optional and only needed for a few tests. It only leaves the dependency on IlmBase for now.

I'm adjusting the title.